package.json

```diff
@@ -44,15 +44,18 @@
     "renameschema": "scripts/rename-schema.sh",
     "presite": "npm run data && npm run build:site && npm run build:toc && npm run build:versions && scripts/create-example-pages",
     "site": "bundle exec jekyll serve --incremental",
+    "tsc:src": "tsc -b src/tsconfig.src.json",
```

Comment:
Do we really need tsc:src?

Reply:
We use this in  bunch of commands and also in the contributing guide.